File: app/src/main/java/com/example/paetz/yacguide/database/Sector.java
package com.example.paetz.yacguide.database;

import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;

@Entity
public class Sector {

    @PrimaryKey
    private int id;

    private float nr;
    private String name;
    private int parentId;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public float getNr() {
        return nr;
    }

    public void setNr(float nr) {
        this.nr = nr;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getParentId() {
        return parentId;
    }

    public void setParentId(int parentId) {
        this.parentId = parentId;
    }
}

File: app/src/main/java/com/example/paetz/yacguide/database/Sector.kt
package com.example.paetz.yacguide.database

import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

@Entity
class Sector {

    @PrimaryKey
    var id: Int = 0

    var nr: Float = 0f
    var name: String? = null
    var parentId: Int = 0
}
